packages/ccdi-models/src/metadata/field/unowned.rs

```diff
@@ -310,6 +310,15 @@ pub mod sample {
         ccdi_cde as cde
     );
 
+    unowned_field!(
+        LibrarySourceMaterial,
+        field::unowned::sample::LibrarySourceMaterial,
+        cde::v1::sample::LibrarySourceMaterial,
+        cde::v1::sample::LibrarySourceMaterial,
+        cde::v1::sample::LibrarySourceMaterial::NotReported,
```

Comment:
@e-t-k @claymcleod I am not sure how this unowned part works. Can I just use one field at choice? I just replaced Other with NotReported for now.

Reply:
Yeah, this is just for the example that is generated. This field is fine.